arcgis-ios-sdk-samples/Geometry/List transformations/README.md

```diff
@@ -25,7 +25,7 @@ When the user taps on a transformation in the list, the selected transformation
 * AGSGeographicTransformationStep
 * AGSGeometryEngine.project
 
-## [Offline data](id:offline_data)
```

Comment:
This isn't needed for the link to work? Cool!

Reply:
The link never worked for me, so I removed it and in the next commit the link reference.